F4transkript/f4transkript.munki.recipe

```diff
@@ -31,12 +31,30 @@
 			<key>name</key>
 			<string>%NAME%</string>
 			<key>unattended_install</key>
+			<true/>
 			<key>blocking_applications</key>
 			<array>
 				<string>F5transkript</string>
 				<string>f4transkript</string>
 			</array>
-			<true/>
+			<key>preinstall_script</key>
```

Comment:
This isn't needed if munki can detect the running applications using the information in `blocking_applications`. See https://github.com/munki/munki/wiki/Blocking-Applications